source/extensions/filters/http/custom_response/custom_response_filter.cc

```diff
@@ -22,8 +22,9 @@ Http::FilterHeadersStatus CustomResponseFilter::decodeHeaders(Http::RequestHeade
   // Note that the original request header map is NOT carried over to the
   // redirected response. The redirected request header map does NOT participate
   // in the custom response framework.
-  auto filter_state = encoder_callbacks_->streamInfo().filterState()->getDataReadOnly<Policy>(
-      "envoy.filters.http.custom_response");
+  auto filter_state =
+      encoder_callbacks_->streamInfo().filterState()->getDataReadOnly<CustomResponseFilterState>(
+          "envoy.filters.http.custom_response");
```

Comment:
nit: I suggest moving the name to a `static constexpr absl::string_view` field (defined in `CustomResponseFilterState`), that can be read by the Policy.

Reply:
Done